packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart

```diff
@@ -53,12 +53,12 @@ class InspectorBreadcrumbNavigator extends StatelessWidget {
   }
 
   List<_InspectorBreadcrumbData> _generateBreadcrumbs(
-    List<InspectorTreeRow> rows,
+    List<InspectorTreeNode> nodes,
   ) {
-    final List<_InspectorBreadcrumbData> items = rows.map((row) {
+    final List<_InspectorBreadcrumbData> items = nodes.map((node) {
       return _InspectorBreadcrumbData.wrap(
-        row: row,
-        isSelected: row == rows.safeLast,
+        node: node,
+        isSelected: node  == nodes.safeLast,
```

Comment:
pull `nodes.safeLast` out into a var before calling List.map so that it doesn't need to be called for every item in the list.